manifests/repo.pp

```diff
@@ -5,126 +5,160 @@
 # This was used in favor of the yumrepo type, which cannot
 # manage files in that directory.
 #
-# example
-#  yum::repo { 'redhat-base':
-#    gpgcheck => '1',
-#  }
+# === Parameters
```

Comment:
This should be under `@summary` Check this out

https://github.com/sensu/sensu-puppet/blob/master/manifests/init.pp#L1-L4

see it in action at

http://sensu.github.io/sensu-puppet/doc/puppet_classes/sensu.html

Reply:
Thanks, didn't know that. Updated accordingly.